File: src/01-easy/the-descent/solution.java
import java.util.Scanner;
class Player {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        while (true) {
            int max = -1, idx = 0;
            for (int i = 0; i < 8; i++) {
                int h = s.nextInt();
                if (h > max) { max = h; idx = i; }
            }
            System.out.println(idx);
        }
    }
}

File: src/01-easy/the-descent/solution.m
#import <Foundation/Foundation.h>
int main() {
    while (1) {
        int max = -1, idx = 0;
        for (int i = 0; i < 8; i++) {
            int h; scanf("%d", &h);
            if (h > max) { max = h; idx = i; }
        }
        printf("%d\n", idx);
    }
    return 0;
}
